spec/factories/valid_draft_factories.rb

```diff
@@ -815,7 +819,7 @@
           'Eon' => 'test 3 Eon text 1'
         }]
       }],
-      'ISOTopicCategories' => %w(farming climatologyMeteorologyAtmosphere health)
+      'ISOTopicCategories' => %w[farming climatologyMeteorologyAtmosphere health]
```

Comment:
Yeah, looks like it was changed a couple of months ago. I updated everything and now I don't see the warning